vite.base.config.ts

```diff
@@ -40,6 +40,7 @@ export function getBaseViteConfig(
     esbuild: {
       banner,
       legalComments: "none",
+      sourcemap: "inline",
```

Comment:
The `vite` file is not for the `vitest`, rather it's for the bundle packages. I am surprised changing any of this effects debugging.